test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalDeleteTests.cs

```diff
@@ -108,13 +111,15 @@ public async Task GivenMultipleMatchingResources_WhenDeletingConditionallyWithMu
 
         [InlineData(true)]
         [InlineData(false)]
-        [Theory]
+        [SkippableTheory]
         public async Task Given50MatchingResources_WhenDeletingConditionallyWithMultipleFlag_TheServerShouldDeleteSuccessfully(bool hardDelete)
         {
             var identifier = Guid.NewGuid().ToString();
 
             await Task.WhenAll(Enumerable.Range(1, 50).Select(_ => CreateWithIdentifier(identifier)));
-            await ValidateResults(identifier, 50);
+
+            var countOfCreated = await GetResourceCount(identifier);
```

Comment:
>countOfCreated [](http://example.com/codeflow?start=16&length=14)

why not use this count for deletion and assert?

Reply:
Good point. Though the tests are run with Session consistency, I can just anticipate this will sometimes not be true if something goes wrong during setup (i.e. the token doesn't get passed along properly, then it becomes a timing thing until the data from the session is readable by all).